docs/interface.rst

```diff
@@ -385,9 +385,24 @@ The follow event kinds are understood (see ``status.py`` for more details on the
 
 - ``"download-finished"``: same as upload version.
 
+- ``"invite-created"``: A new invite is created. All invite events also have a ``folder`` indicating which folder they pertain to, a ``uuid``, ``participant-name`` indicating the invitee and ``mode`` (``"read-only"`` or ``"read-write"``).
```

Comment:
Other RESTful endpoints (beneath `/experimental/magic-folder/<folder>/invite-*`) use `id` for the UUID whereas these use `uuid`. For consistency's sake -- and to signal to consumers of these APIs that the values are interchangeable/compatible -- consider using the same key/field name throughout.